docs/tutorial/procedure.rst

```diff
@@ -333,4 +333,9 @@ Now that you have a background on how to use the different features of the Proce
         worker.join(timeout=3600) # wait at most 1 hr (3600 sec)
         log.info("Finished the measurement")
 
+The parentheses in the :code:`COLUMN` entries indicate the physical unit of the data in the corresponding column, e.g. :code:`'Voltage Std (V)'` indicates Volts. If you want to indicate a unitless value, e.g. Mach number, you can use `(1)` instead. Combined units like `(m/s)` or the long form `(meter/second)` are also possible. The class :class:`Results` ensures, that the data is stored in the correct unit, here Volts. For example a :python:`pint.Quantity` of 500 mV will be stored as 0.5 V. A string will be converted first to a `Quantity` and a mere number (e.g. float, int, ...) is assumed to be already in the right unit (e.g 5 will be stored as 5 V).
```

Comment:
I can't see a "suggestions" button, so this way
> If you want to indicate a **unitless** value

please use "dimensionless" for such. (https://en.wikipedia.org/wiki/Dimensionless_quantity)